content/en/serverless/azure_app_services/_index.md

```diff
@@ -66,9 +66,9 @@ Capabilities:
 [4]: https://app.datadoghq.com/serverless/azure/app-service-plan
 [5]: /logs/guide/azure-logging-guide/
 [6]: /integrations/azure/#setup
-[7]: /serverless/azure_app_services/azure_app_services_windows?tab=net#setup
-[8]: /serverless/azure_app_services/azure_app_services_windows?tab=java#setup
-[9]: /serverless/azure_app_services/azure_app_services_linux?tab=nodenetphppython
+[7]: /serverless/azure_app_services/azure_app_services_windows?tab=net
+[8]: /serverless/azure_app_services/azure_app_services_windows?tab=java
+[9]: /serverless/azure_app_services/azure_app_services_linux?tab=net
```

Comment:
```suggestion
[9]: /serverless/azure_app_services/azure_app_services_linux?tab=nodejs
```

Reply:
They look correct to me.
```
|Windows|Node.js|[Windows Node.js setup][13]|
|Linux|.NET|[Linux .NET setup][9]|
```

They're just out of order. I'll fix that...

Reply:
OK, I'll go over all the links. I see now the other issues you mentioned elsewhere 😅 .

Reply:
I redid all the links. 🤞🏽